Tests/SonarScanner.MSBuild.Shim.Test/ScannerEngineInputGeneratorTest.GenerateResult.cs

```diff
@@ -66,121 +66,45 @@ public void GenerateResult_ValidFiles()
 
         // One valid project info file -> file created
         AssertScannerInputCreated(result);
-        result.ScannerEngineInput.ToString().Should().BeIgnoringLineEndings(
+
+        result.ScannerEngineInput.ToString().Should().ContainAll([
```

Comment:
We don't care about the `ToString` part here. We want to assert the values. 
Use `ScannerEngineInputReader` and assert the values using that